From 25c7a13985473ff9714245eaa602225b1a0f9ecb Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.de>
Date: Wed, 25 Mar 2020 12:23:11 +0100
Subject: [PATCH] 90nvdimm: include nvdimm keys in initrd

This is necessary to actually unlock NVDIMM keys during boot.
---
 modules.d/90nvdimm/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90nvdimm/module-setup.sh b/modules.d/90nvdimm/module-setup.sh
index 6d58efd7..6d73d215 100755
--- a/modules.d/90nvdimm/module-setup.sh
+++ b/modules.d/90nvdimm/module-setup.sh
@@ -27,5 +27,5 @@ installkernel() {
 
 # called by dracut
 install() {
-    inst_multiple -o ndctl
+    inst_multiple -o ndctl /etc/ndctl/keys/tpm.handle /etc/ndctl/keys/*.blob
 }

